src/Configurator/DockerComposeConfigurator.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Configurator;
+
+use Symfony\Flex\Recipe;
+
+/**
+ * Adds services and volumes to docker-compose.yaml file.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DockerComposeConfigurator extends AbstractConfigurator
+{
+    public function configure(Recipe $recipe, $config): void
+    {
+        $dockerCompose = getcwd().'/docker-compose.yaml';
+        if (!file_exists($dockerCompose) || $this->isFileMarked($recipe, $dockerCompose)) {
+            return;
+        }
+
+        $this->write('Adding Docker Compose entries');
+
+        $lines = [];
+        foreach (file($dockerCompose) as $line) {
+            $lines[] = $line;
+            if (!preg_match('/^[\'"]?([a-zA-Z0-9]+)[\'"]?:\s*$/', $line, $matches) || !isset($config[$matches[1]])) {
```

Comment:
If i will add a comment at the end of a line with `"services":` in my `docker-compose.yml`, there will be two `services` sections there after applying `$config`